portal-web/test/functional/com/liferay/portalweb/paths/portlet/pageaudit/PageAudit.path

```diff
@@ -0,0 +1,34 @@
+<html>
+<head>
+<title>PageAudit</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">PageAudit</td></tr>
+</thead>
+<tbody>
+<tr>
+	<td>CHECK_ISSUES_TEXT</td>
+	<td>//*[contains(@class, 'font-weight-semi-bold') and contains(@class, 'c-mb-2')]</td>
```

Comment:
this class name is right and not will change (random feeling) ?

> c-mb-2

Reply:
Hope not, but just in case I will delete it and modify the path

Reply:
I'm not super fan of getting the CSS classes for the paths 🤷‍♀️  But hopefully it will work. Let's see when the code gets merged.